js/bank.js

```diff
@@ -126,4 +121,20 @@ function isPositive(amount) {
   }
 }
 
-start(100);
+function message() {
```

Comment:
I like that you pulled this into its own method. I'd like to see a more specific name.

Reply:
I will change it to closingMessage!